draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -914,6 +914,51 @@ The work on this draft was started at OAuth Security Workshop 2022 in Trondheim,
 
 TBD
 
+## Media Type Registration
+
+This section requests registration of the "application/sd-jwt" media type [@RFC2046] in
```

Comment:
@OR13 @tlodderstedt when we discussed in person, how did we say this one media type will be used for both issuance and presentation of SD-JWT? Those will have different processing rules (section 6) and the latter might have HB JWT too...

Reply:
I don't recall, but if it is in the protected header, and must be disclosed, it can't be changed after issuance.

There won't be any integrity protection on a "presentation format token" unless it is signed over by the holder, so as you say, holder binding JWT probably wants a specific sub type... something like:

`application/hb-sd-jwt`

Reply:
The media type is always the same (as it is signed by the issuer). The additional hb JWT does not seem to have a dedicated `typ` yet, but I think it should have one. `application/hb+jwt` is reasonable to me. 

What do the co-authors think? @bc-pi @Sakurann @danielfett

Reply:
`application/hb+jwt` is reasonable to me too but I'm not sure the HB JWT needs to be typed.

Reply:
@bc-pi yea, it does not need to be typed, unless you want to ensure header or claimset members are processed a specific way... the `nonce` seems like it might be one of those... not sure.

Reply:
ok, so do I understand that the same application/sd-jwt will be used regardless whether it is issuance and presentation and the receiver is expected to know which combined format it is, based on the context, protocol?

Reply:
Yes @Sakurann, I believe so.